pkg/generic/httpthrift_codec.go

```diff
@@ -94,11 +116,20 @@ func (c *httpThriftCodec) Marshal(ctx context.Context, msg remote.Message, out r
 
 	inner := thrift.NewWriteHTTPRequest(svcDsc)
 	inner.SetBinaryWithBase64(c.binaryWithBase64)
+	inner.SetEnableDynamicgo(c.enableDynamicgoReq)
+	if c.enableDynamicgoReq {
```

Comment:
You can define a wrapped config with convOpts to avoid pass `enableDynamicgoReq` and `convOpts` twice.

Reply:
fixed